client/src/components/EditHistory.js

```diff
@@ -1,4 +1,4 @@
-import { Button } from "@blueprintjs/core"
+import { Button, Intent } from "@blueprintjs/core"
```

Comment:
I think we should completely use Bootstrap buttons to be consistent in UI. Using two different button types from two different UI libraries does not make sense to me.

Reply:
Sorry for not bringing that up before you start fixing buttons.

Reply:
I was halfway replacing those and then thought it better to leave this as a separate action. So put that on the TODO list.